extensions/blocks/tiled-gallery/tiled-gallery.php

```diff
@@ -27,11 +27,7 @@ class_exists( 'Jetpack_Photon' ) && Jetpack::is_module_active( 'photon' )
 	 * @return string
 	 */
 	function jetpack_tiled_gallery_load_block_assets( $attr, $content ) {
-		$dependencies = array(
-			'lodash',
-			'wp-i18n',
-			'wp-token-list',
-		);
+		$dependencies = array();
```

Comment:
`load_assets_as_required` already defaults to empty array so I think we can remove this.

https://github.com/Automattic/jetpack/blob/3283ec371c6e8ed01b3bbf021a5d4f5aac235b1e/class.jetpack-gutenberg.php#L455

Reply:
Cleaned it up in ff4806a6070c6432d78798b253a513094f1993a7